deploy/fabfile.py

```diff
@@ -33,26 +32,58 @@
 home = "/home/site"
 log = "%s/dashboard.log" % home
 current = "%s/%s/dashboard" % (home, environment)
+now = time.strftime("%Y-%m-%d", time.localtime())
 
 # principal command to run upon update
-command = "cd %s && git pull && git submodule update && jekyll build >> %s" % (current, log)
+staging = "cd %s && git pull && git submodule update --remote && ./_data/import-public.rb && git add _data/projects.yml && git commit -m 'Update data for %s' && git push && ./go build >> %s" % (current, now, log)
```

Comment:
Okay, I think this is all ready to go with 78566d0, though Travis is failing so I'm looking into that.